packages/commonwealth/client/scripts/views/components/EditProfile/EditProfile.tsx

```diff
@@ -0,0 +1,433 @@
+                <CWText type="caption" fontWeight="medium">
+                  Profile image
+                </CWText>
+                <CWText type="caption" className="description">
+                  Select an image from your files to upload
+                </CWText>
+                <div className="image-upload">
+                  <AvatarUpload
+                    scope="user"
+                    account={account}
+                    uploadStartedCallback={() =>
+                      setIsUploadingProfileImage(true)
+                    }
+                    uploadCompleteCallback={(files) => {
+                      setIsUploadingProfileImage(false);
+                      files.forEach((f) => {
+                        if (!f.uploadURL) return;
+                        const url = f.uploadURL.replace(/\?.*/, '').trim();
+                        setAvatarUrl(url);
+                      });
+                    }}
+                  />
+                </div>
+              </div>
+              <CWTextInput
+                fullWidth
+                placeholder="Enter your user name"
+                label="Username"
+                // TODO: unique username like in PersonalInformationStep?
+                // TODO: username generator like in PersonalInformationStep?
+                name="username"
+                hookToForm
+              />
+              <CWTextInput
+                fullWidth
+                placeholder="Add an email address"
+                label="Email"
+                name="email"
+                hookToForm
+              />
+              <ReactQuillEditor
+                label="Bio"
+                className="editor"
+                hookToForm
+                name="bio"
+              />
+              <CWDivider />
+              <LinksArray
+                label="Social links"
+                addLinkButtonCTA="+ Add social link"
+                links={links.map((link) => ({
+                  ...link,
+                  customElementAfterLink:
+                    link.value && getLinkType(link.value, 'website') ? (
+                      <CWTag
+                        label={getLinkType(link.value, 'website')}
+                        type="group"
+                        classNames="link-type"
+                      />
+                    ) : (
+                      <></>
+                    ),
+                }))}
+                onLinkAdd={onLinkAdd}
+                onLinkUpdatedAtIndex={onLinkUpdatedAtIndex}
+                onLinkRemovedAtIndex={onLinkRemovedAtIndex}
+                canAddLinks={links.length <= 5}
+              />
+            </ProfileSection>
+            <ProfileSection
+              title="Personalize Your Profile"
+              description="Express yourself through imagery."
+            >
+              <CWText fontWeight="medium">Add a background image </CWText>
+              {/* TODO: add option to remove existing image */}
```

Comment:
Another potential enhancement, this behavior is not yet implemented.

I feel we would want this, will be creating a new ticket for this.